server/aws-lsp-codewhisperer/src/language-server/agenticChat/constants/constants.ts

```diff
@@ -18,6 +18,7 @@ export const GENERATE_ASSISTANT_RESPONSE_INPUT_LIMIT = 500_000
 // 200K tokens * 3.5 = 700K characters, intentionally overestimating with 3.5:1 ratio
 export const MAX_OVERALL_CHARACTERS = 700_000
 export const COMPACTION_CHARACTER_THRESHOLD = 0.7 * MAX_OVERALL_CHARACTERS
+export const MID_LOOP_COMPACTION_HANDOFF_PROMPT = `CONTEXT HANDOFF: Previous conversation was compacted. Continue with user's request: `
```

Comment:
where did we get this?

Reply:
I just wrote this prompt myself. Do you think we need to discuss crafting a prompt for this?